pkg/vrct/vrctFs/vrct.go

```diff
@@ -141,140 +172,299 @@ func mergePrototypes(prototypesDirPath, destPath string) error {
 			}
 			continue
 		}
-		fmt.Printf("[WARNING] Unrecognized file %s in %s\n", prototypeName, prototypesDirPath)
 	}
 
 	return nil
 }
 
-type FsRequirement struct {
-	// In simpler words - how this rule appeared here
-	ruleStackTrace   []string
-	checkRequirement func() bool
+func (p *FilePrototype) mergeLayers() (PrototypeLayer, error) {
+	switch p.FileType {
+	case TextFile:
+		return p.mergeTextLayers()
+	default:
+		return p.mergeConfigLayers()
```

Comment:
Are you sure it should be putted into default case?

Reply:
CreateLayer handles unsupported file type